libraries/load-themed-styles/gulpfile.js

```diff
@@ -1,9 +1,40 @@
 'use strict';
 
-let build = require('@microsoft/node-library-build');
+const build = require('@microsoft/node-library-build');
+const path = require('path');
+
+build.tscCmd = 'tsc-commonjs';
+
+const tscAmdTask = new build.TscCmdTask();
+tscAmdTask.name = 'tsc-amd';
+tscAmdTask.setConfig({
+  customArgs: [
+    '--outDir', './lib-amd',
+    '--module', 'amd'
+  ]
+});
+
+const tscEsnextTask = new build.TscCmdTask();
+tscEsnextTask.name = 'tsc-es6';
+tscEsnextTask.setConfig({
+  customArgs: [
+    '--outDir', './lib-es6',
+    '--module', 'esnext'
+  ]
+});
+
+build.defaultTasks = build.task(
+  'default',
+  build.parallel(
+    build.defaultTasks,
+    tscAmdTask,
+    tscEsnextTask
+  )
+);
 
 build.setConfig({
-  libAMDFolder: 'lib-amd'
+  libAMDFolder: 'lib-amd',
+  libES6Folder: 'lib-es6'
```

Comment:
Do we need this, if we're manually creating the tasks above?

Reply:
yes

---
In reply to: [213862484](https://github.com/Microsoft/web-build-tools/pull/798#discussion_r213862484) [](ancestors = 213862484)